integration/spark2/src/main/scala/org/apache/spark/sql/optimizer/CarbonFilters.scala

```diff
@@ -395,4 +391,14 @@ object CarbonFilters {
       case _ => expressions
     }
   }
+
+  def getPartitions(partitionFilters: Seq[Expression],
```

Comment:
I think it is better to do this logic in `pruneFilterProject` directly, this function is not needed

Reply:
This function will be further used in other places for other features like compaction.

Reply:
Can you add comment for this function

Reply:
move `partitionFilters` to next line